history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/TaskanaHistoryEngineImpl.java

```diff
@@ -108,6 +112,16 @@ protected SqlSessionManager createSqlSessionManager() {
     Configuration configuration = new Configuration(environment);
 
     // register type handlers
+    try (Connection connection = taskanaEngineConfiguration.getDatasource().getConnection()) {
+      if (DB.isOracleDb(DB.getDatabaseProductName(connection))) {
+        // Use NULL instead of OTHER when jdbcType is not specified for null values,
+        // otherwise oracle driver will chunck on null values
+        configuration.setJdbcTypeForNull(JdbcType.NULL);
+        configuration.getTypeHandlerRegistry().register(String.class, new StringTypeHandler());
+      }
+    } catch (SQLException sqle) {
+      throw new SystemException("Issue detecting DB product name", sqle);
+    }
```

Comment:
you forgot to use `OracleSqlSessionFactory` for the SqlSessionFactory ;)

Reply:
thank you